base/interactiveutil.jl

```diff
@@ -571,11 +571,10 @@ end
 downloadcmd = nothing
```

Comment:
you're changing the download function to shell out to something, the value of what you're shelling out to should be saved in the global that exists for that purpose